soc/projects/numerics.md

```diff
@@ -82,6 +93,8 @@ and boundary cases.
 
 **Expected Results**: New and faster methods for evaluating properties of special functions.
 
+**Mentors:** [Alex Arslan](https://github.com/ararslan)
```

Comment:
Also I'm probably not the best person to consult in regards to developing the special functions code. That'd probably be @simonbyrne, if he's able.